change/@graphitation-graphql-codegen-supermassive-typed-document-node-plugin-a48e73bf-eef0-42f6-8905-545df5db8728.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "add lazy ast mode",
```

Comment:
Those goes to changelog, so deserves a bit more data. Mention the breaking bits too (removed V2/BOTH modes, something else?)

Reply:
done